newIDE/app/src/GameDashboard/GameDetailsDialog.js

```diff
@@ -146,7 +149,9 @@ export const GameDetailsDialog = ({
     [loadPublicGame]
   );
 
-  const updateGameFromProject = async () => {
+  const updateGameFromProject = async (
+    apiOnlyPublicGameInfo: PartialGameChanges
```

Comment:
now that you've changed the type name, maybe also change the variable name? `partialGameChanges`?